iroha/src/crypto.rs

```diff
@@ -52,43 +55,27 @@ impl TryFrom<Vec<u8>> for PublicKey {
 }
 
 /// Private Key used in signatures.
-#[derive(Clone)]
+#[derive(Clone, Debug, Deserialize, PartialEq)]
 pub struct PrivateKey {
-    inner: [u8; 64],
-}
-
-impl Debug for PrivateKey {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        fmt::Debug::fmt(&&self.inner[..], f)
-    }
+    inner: Vec<u8>,
```

Comment:
The same suggestion here...
```suggestion
    #[serde(flatten)]
    inner: Vec<u8>,
```

Reply:
`Error("can only flatten structs and maps", line: 1, column: 197)`
Looks like we can't flatten Vec - I assume we will need to work on it in https://jira.hyperledger.org/browse/IR-745

Reply:
@humb1t oh, ok. There is also `#[serde(transparent)]` for a struct itself. Can you try it?

Reply:
@vmarkushin I will try to play with it in IR-745